app/code/Magento/CatalogUrlRewrite/Observer/AfterImportDataObserver.php

```diff
@@ -340,7 +340,7 @@ private function setStoreToProduct(Product $product, array $rowData)
      */
     private function addProductToImport($product, $storeId)
     {
-        if ($product->getVisibility() == (string)Visibility::getOptionArray()[Visibility::VISIBILITY_NOT_VISIBLE]) {
```

Comment:
`$product->getVisibility()` returns data from  csv file like 'Catalog, Search' or 'Not visible individually' and `Visibility::VISIBILITY_NOT_VISIBLE` is type of INT that will never pass because `'Not visible individually'  ` !== 1
Could you please explain ?